README.md

```diff
@@ -84,7 +84,9 @@ If your application is running outside of Google Cloud Platform, such as locally
   2. Copy the key somewhere your application can access it. Be sure not to expose the key publicly.
   3. Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the full path to the key. The debug agent will automatically look for this environment variable.
```

Comment:
Rationale? [google-cloud](http://npm.im/google-cloud) modules suggest config as the default mechanism.